Dockerfile

```diff
@@ -1,5 +0,0 @@
-FROM registry.access.redhat.com/ubi8/ubi:8.1
```

Comment:
I think we need to keep this to support [the local `deploy/` manifests][1], for the "I want my local test cluster to build me a graph-data image".

[1]: https://github.com/openshift/cincinnati-graph-data/blob/140221b0062305321b851ada0cd7478da08a348a/deploy/2-build-config.yaml

Reply:
Pushed the content of dockerfile into `bc.spec.source.dockerfile`.
I really want to make the top level files as few as possible.
Otherwise, a user could wonder why it stays here, is it a part of grapha-data?